iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -61,12 +61,12 @@ void publisher()
     iox::popo::Publisher<TransmissionData_t> publisher({"Single", "Process", "Demo"}, publisherOptions);
 
     uint64_t counter{0};
-    std::string greenRightArrow("\033[32m->\033[m ");
+    constexpr const char greenRightArrow[] = "\033[32m->\033[m ";
```

Comment:
According to our coding guidelines all compile time constants have to be written in UPPER_CAMEL_CASE.
Could you please rename it to `GREEN_RIGHT_ARROW`.

The same goes for all the other arrows.

Reply:
mea culpa. I think I introduced it ... and no reviewer found it ... shame on them ;)

Reply:
changed to UPPER_CAMEL_CASE